lib/sql.js

```diff
@@ -82,29 +88,17 @@ class SQL {
 	/**
 	* Count the number of entry in a table
 	* @param {string} param - query parameter
-	* @param {number} limit - limit
-	* @param {number} offset - offset
```

Comment:
I think the param type is `Number` instead of `number`, `String` instead `string`.